opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -245,9 +245,9 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 14;
 
-  // An optional number of child spans that were generated while this span
-  // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 15;
+  // child_span_count is an optional number of local child spans that were generated while this
+  // span was active. If set, allows an implementation to detect missing child spans.
+  uint32 child_span_count = 15;
```

Comment:
How do we know if 0 means it was real 0 or not set?

Reply:
It will not be possible. We had this discussion on this field here https://github.com/open-telemetry/oteps/pull/59/files#r338174095

If you believe the distinction between 0 and "not set" is important then this change will not work. Can you think of a use case where it matters?

Reply:
I know that at least with protobuf v2 and Java, for each field a `hasMyField` method was generated you could use to check if it was received.

Reply:
@Oberon00 not true for v3 :(

Reply:
@Oberon00 protobuf v3 removed the distinction: https://developers.google.com/protocol-buffers/docs/proto3#default
>Note that for scalar message fields, once a message is parsed there's no way of telling whether a field was explicitly set to the default value (for example whether a boolean was set to false) or just not set at all: you should bear this in mind when defining your message types.

Reply:
This field is used to determine completion of a trace. If it is 0 the system cannot determine if this is a real leaf or the value is not set.

Reply:
it's only _local_ leaves though, there's no way of knowing about remote leaves... so I'm not sure how this helps with knowing if the trace is finished...

Reply:
Another option is to reserve MAX_INT for undefined.

Reply:
oh, I like that one.

Reply:
I'd suggest -1 is a better value for undefined. However, I think @lizthegrey is right--perhaps this should be named "internal_child_count" (for `SpanKind==INTERNAL`) or "local_child_count".

It would also make sense, to me, to have a count of the number of times a context is extracted for a remote child, but I'm not sure it would be reliable way to test for completion since the child span could fail or be disregarded in some way.

Reply:
I think we should really think about this remote/local child problem. Discussion had already started in the original OTEP https://github.com/open-telemetry/oteps/pull/59/files#r336737075.